test.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
 <style>
-body {color: black;}
+div {color: black;}
```

Comment:
Maybe this can improve the code style
```suggestion
div {
	color: black;
}
```